config/env/default.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
     title: 'MEAN.JS',
     description: 'Full-Stack JavaScript with MongoDB, Express, AngularJS, and Node.js',
     keywords: 'mongodb, express, angularjs, node.js, mongoose, passport',
-    googleAnalyticsTrackingID: process.env.GOOGLE_ANALYTICS_TRACKING_ID || 'GOOGLE_ANALYTICS_TRACKING_ID'
+    googleAnalyticsTrackingID: process.env.GOOGLE_ANALYTICS_TRACKING_ID || 'GOOGLE_ANALYTICS_TRACKING_ID',
+    domain: process.env.DOMAIN
```

Comment:
`domain: process.env.DOMAIN || 'localhost'` won't work since it has to include the port.

For example it could (and should) be used to assemble the URL to reset the password. This url has to include the port.